stellar/its.js

```diff
@@ -0,0 +1,63 @@
+const { Command } = require('commander');
+const { Contract, nativeToScVal} = require('@stellar/stellar-sdk');
+
+const { saveConfig, loadConfig, addOptionsToCommands, getChainConfig } = require('../common');
+const { addBaseOptions, getWallet, broadcast } = require('./utils');
+
+async function setTrustedAddress(wallet, _, chain, contractConfig, args, options) {
+    const contract = new Contract(contractConfig.address);
+    const [chainName, trustedAddress] = args;
+    const callArgs = [chainName, trustedAddress].map(nativeToScVal);
+
+    const operation = contract.call('set_trusted_address', ...callArgs);
+
+    await broadcast(operation, wallet, chain, 'Trusted Address Set', options);
+}
+
+async function removeTrustedAddress(wallet, _, chain, contractConfig, arg, options) {
+    const contract = new Contract(contractConfig.address);
+    const callArg = nativeToScVal(arg, { type: 'string' });
+
+    const operation = contract.call('remove_trusted_address', callArg);
+
+    await broadcast(operation, wallet, chain, 'Trusted Address Removed', options);
+}
+
+async function mainProcessor(processor, args, options) {
+    const config = loadConfig(options.env);
+    const chain = getChainConfig(config, options.chainName);
+
+    const wallet = await getWallet(chain, options);
+
+    if (!chain.contracts?.interchain_token_service) {
+        throw new Error('Interchain Token Service package not found.');
+    }
+
```

Comment:
add a `prompt(..)` here for consistency (see existing usage)

Reply:
I see prompts only on `deploy` and `upgrade` from `deploy-contract.js`. Neither `gateway.js` nor `operator.js` use prompts. What do you want it to say? Just ask the user if they want to continue?